### PR TITLE
[IMP] Markdown format in readme

### PR DESCRIPTION
**Pull request by forvas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
+# Py_pg_tools
 
     A powerful tool to use PostgreSQL in an easier and faster way. It allows
     the possibility to:
 
-    - Change the owner of a database and all of its tables.
-    - Make a custom backup of a PostgreSQL database or a whole cluster.
-    - Delete a PostgreSQL database.
-    - Get some information about some PostgreSQL databases, users or its
+    * Change the owner of a database and all of its tables.
```
Comment (alejandrosantana) [inline README.md:7]: Dashes (-) work well in markdown for bullets. In fact, I find them less dangerous than asterisks, as they are used for italic (single) or bold (double).

File: README.md [hunk 2]
```diff
@@ -1,36 +1,36 @@
 
-PY_PG_TOOLS:
+# Py_pg_tools
```
Comment (alejandrosantana) [inline README.md:3]: There could be a level 2 title such as `## Description`

File: README.md [hunk 3]
```diff
@@ -1,36 +1,36 @@
...
```
Comment (alejandrosantana) [inline README.md:2]: Module name is in lowercase, isn't it?

File: README.md [hunk 4]
```diff
@@ -1,36 +1,36 @@
...
     If you want to see more information about a specific module, write its
     characteristic letter and then the argument -h or --help. For example,
     in case you need information about how to make a backup (supposing you are
     in the same directory of py_pg_tools.py):
 
-    - python3 py_pg_tools.py B -h
+    * python3 py_pg_tools.py B -h
 
-IMPORTED PACKAGES:
+# Imported packages:
```
Comment (alejandrosantana) [inline README.md:33]: This should probably be second level title (also the rest): `## Imported packages:`

Comment (forvas) [inline README.md:33]: https://github.com/anubia/py_pg_tools/commit/d0acf487554715bacec8d5bbbd0f2ef6184c02e2

2015-04-20 17:32 GMT+02:00 Alejandro Santana notifications@github.com:

> In README.md
> https://github.com/anubia/py_pg_tools/pull/8#discussion_r28699057:
> 
> > -IMPORTED PACKAGES:
> > +# Imported packages:
> 
> This should probably be second level title (also the rest): ## Imported
> packages:
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/anubia/py_pg_tools/pull/8/files#r28699057.

## 

## 

_Juan Formoso Vasco_
jfv@anubia.es
· · · · · · · · · · · · · · · · · · · · · · · · · · · · ·

ANUBÍA, soluciones en la nube, S.L.

_Colón 26, planta 3, oficina 1_

36201, Vigo (Pontevedra)

www.anubia.es

[image: anubía_logo_2014_i_o_140x140.png][image:
odoo_logo_odoo_ready_140x140.png] http://www.anubia.es/

**Pull request merged**